packages/shared/src/lib/labels.ts

```diff
@@ -33,7 +33,7 @@ export const labels = {
   },
   devcard: {
     generic: {
-      shareText: `Excited to share that I've created my personalized devcard on daily.dev, highlighting my favorite tech topics and publications. Create yours now!`,
+      shareText: `Check out my #DevCard by @dailydotdev! Flex yours (if it's flex worthy)`,
```

Comment:
This would still not add the actual image, wasn't sure, but that was how I interpreted the message?

Reply:
Oh wait maybe it;'s a different prop?